hamster_gtk/overview/dialogs.py

```diff
@@ -109,6 +109,9 @@ def refresh(self):
         self._grouped_facts, self._totals = self._group_facts()
 
         helpers.clear_children(self.main_box)
+        if self._charts:
```

Comment:
I am not sure I understand why this works. ``helpers.clear_children(self.main_box)`` should do exactly that the new lines do: call ``destroy`` on all its children. And whenever ``self._charts`` exists its also a child of ``main_box``.
Can you please elaborate?
I can however, confirm that you fix solves the issue.